src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,105 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+  Background:
+  
+
+  @ignore
+  Scenario: Ensure clients can GET information when querying by valid date
+    Given a school exists
+            | schoolId   | nameOfInstitution                             | gradeLevels               | educationOrganizationCategories          |
+            | 255901001 | Grand Bend High School                         | [ "Tenth grade" ]         | [ "School" ]                             |    
+    And there is a valid AcademicWeek using a valid BeginDate
+            | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+            | string         | 2024-04-09 | 2024-04-09 | 0                      |
+    When sending a GET request to "/ed-fi/academicWeeks?beginDate=<beginDate>"  
+            """
+            """
+    Then it should respond with 200
+    And response contains "weekIdentifier: "string" "
+
+  @ignoreScenario 
+  Scenario: Ensure clients can't GET information when querying by invalid date
+    Given a school exists
+            | schoolId   | nameOfInstitution                             | gradeLevels               | educationOrganizationCategories          |
+            | 255901001 | Grand Bend High School                         | [ "Tenth grade" ]         | [ "School" ]                             |
+    And there is a valid AcademicWeek using an invalid BeginDate   
+            | weekIdentifier | beginDate | endDate    | totalInstructionalDays |
+            | string         | -04-09    | 2024-04-09 | 0                      |
```

Comment:
The "Given" section must insert valid data into the database. Therefore the `beginDate` above needs to be a real one.

It is on the "When" line that we have an invalid date. In this case you have a perfectly good line, since `<beginDate>` is, in fact, invalid 😁 .

Reply:
Understood.
Applied